Update Courier Documentation - Part 1 (#1301)
umbraco/UmbracoDocs: Add-ons/UmbracoCourier/Developer/Provider-Configuration.md

</fileItemProvider>
```

## Folders
Same as above, but only with folders

```xml
<folderItemProvider>
    <include>
        <folder>~/media/assets/somefolder</folder>
    </include>
</folderItemProvider>
```

## Mediatypes

> Single option to turn of whether allowed/child media types should be added as a dependency or not

Reviewer: ```suggestion
Single option to choose whether allowed/child media types should be added as a dependency or not
```

</mediaTypeItemProvider>
```

## DocumentTypes

- Option to include all allowed templates as a dependency
- Option to include all allowed types as a dependency 

> - Option filter out certain data types.

Reviewer: ```suggestion
- Option to filter out certain data types
```

    <add key="memberpicker">2B24165F-9782-4AA3-B459-1DE4A4D21F60</add>
    <add key="simpleeditor">1251C96C-185C-4E9B-93F4-B48205573CBD</add>
    <add key="truefalse">92897BC6-A5F3-4FFE-AE27-F2E7E33DDA49</add>
    <add key="datepicker">5046194E-4237-453C-A547-15DB3A07C4E1</add>
    <add key="datepickerWithTime">E4D66C0F-B935-4200-81F0-025F7256B89A</add>
    <add key="numeric">2E6D3631-066E-44B8-AEC4-96F09099B2B5</add>
    </ignoredDataTypes>
</documentTypeItemProvider>
```
    
## Media

- Option to include parent nodes as a forced dependency 

> - option to automatically include all children (in case a folder is transferred).

Reviewer: ```suggestion
- Option to automatically include all children (in case a folder is transferred)
```

```xml
<mediaItemProvider>
    <includeChildren>false</includeChildren>
    <includeParents>true</includeParents>
</mediaItemProvider>
```

## Documents

> - Option to include parents as dependencies.

Reviewer: ```suggestion
- Option to include parents as dependencies
```
